backend/coins/btc/transactions/transactions_test.go

```diff
@@ -184,7 +184,6 @@ func (s *transactionsSuite) TestUpdateAddressHistorySingleTxReceive() {
 	)
 	transactions := s.transactions.Transactions(func(blockchainpkg.ScriptHashHex) bool { return false })
 	require.Len(s.T(), transactions, 1)
-	require.Equal(s.T(), tx1, transactions[0].Tx)
 	require.Equal(s.T(), expectedHeight, transactions[0].Height)
```

Comment:
`const expectedHeight = 10` above would make it slightly more reassuring it doesn't change during the test. github doesn't let me comment on unchanged lines :)